technical/deployments-imagineering-session-2.md

```diff
@@ -0,0 +1,58 @@
+# Deployments Imagineering session 2
+Created by Tomas Destefi 02 April 2020
+
+On the last two retros we talked about that our deployment needs a lot of improvements. So 02/04 Me, Brendan and Matt did the session 2 of the Deploy imagineering session.
```

Comment:
In the last two retros we talked about how our deployments need a lot of improvements
```suggestion
In the last two retros we talked about how our deployments need a lot of improvements. So 02/04 Me, Brendan and Matt did the session 2 of the Deploy imagineering session.
```

Reply:
🙈